documentation/smg/08_setting_up_jenkins_ci.md

```diff
@@ -0,0 +1,126 @@
+# Setting Up Jenkins CI
+
+We use machines administered by [ANVIL](https://anvil.softeng-support.ac.uk/)
+to run our continuous integration (CI) jobs. These machines are running
+Jenkins v2.190.3.
+
+We would like to be able to script and commit as much of our CI configuration
+as possible. This makes things re-creatable and locally testable. However,
+there is necessarily some set up that must be done within the GUI in Jenkins.
+It is therefore important that this document is kept up-to-date with the steps
+taken to reach the configuration we use.
+
+We will use the CI for building pull requests and building master every
+evening. Each build will create a Horace package that is shippable to users.
+
+## Overview
+
+Our CI builds follow the following high-level process:
+
+- Jenkins checks out the correct branch
```

Comment:
I'm not sure this is the correct first bullet point; I'd drop it and have "For PR" and "For nightly builds" as the top level entry points and then add a "for both ..." before the "jenkins loads and runs..." bit